src/components/UI/Display/OptionalValue/stories/OptionalValue.stories.tsx

```diff
@@ -0,0 +1,46 @@
+import { Meta, Story } from '@storybook/react/types-6-0';
```

Comment:
```suggestion
import { Meta, Story } from '@storybook/react';
```

This is no longer necessary 👍